packages/rocketchat-lib/server/methods/addUsersToRoom.js

```diff
@@ -34,6 +34,8 @@ Meteor.methods({
 			canAddUser = true;
 		} else if (room.t === 'p' && RocketChat.authz.hasPermission(userId, 'add-user-to-any-p-room')) {
 			canAddUser = true;
+		} else if (room.t === 'r' && userInRoom) {
```

Comment:
I'd rather say we allow all users to `add-user-to-joined-room` - since I expect that not everybody will understand a different behavior between requests and other rooms. @ruKurz @ThomasRoehl wdyt?